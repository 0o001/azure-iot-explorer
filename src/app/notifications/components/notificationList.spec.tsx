/***********************************************************
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License
 **********************************************************/
import * as React from 'react';
import { shallow } from 'enzyme';
import { NotificationList } from './notificationList';

describe('notificationList', () => {
    it('matches snapshot', () => {
        expect(shallow(
            <NotificationList
                hasNew={true}
                notifications={[]}
                dismissNotifications={jest.fn()}
                markAllNotificationsAsRead={jest.fn()}
            />)).toMatchSnapshot();
    });
});
