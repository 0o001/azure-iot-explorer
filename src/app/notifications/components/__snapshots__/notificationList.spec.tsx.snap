// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`notificationList matches snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
